test/cli/testdata/cnspec_scan.ct

```diff
@@ -26,6 +26,7 @@ Flags:
       --inventory-file string       Set the path to the inventory file.
   -j, --json                        Run the query and return the object in a JSON structure.
   -o, --output string               Set output format: compact, csv, full, json, junit, report, summary, yaml (default "compact")
+      --output-target string        Set output target to which the asset report will be sent. Currently only supports AWS SQS topic URLs and local files
```

Comment:
Can we add a cli test that runs a single-check policy, and stores the output on a local disk?